types/index.d.ts

```diff
@@ -80,7 +80,7 @@ declare namespace dayjs {
      *
      * Months are zero indexed, so January is month 0.
      *
-     * Accepts numbers from 0 to 11. If the range is exceeded, it will bubble up to the next year.
```

Comment:
seems "accepts" is better?

Reply:
Because I found that the first sentence of this comment is 'Set the month.', similarly, the first sentence of the other method's comment is 'Get' or 'Set', so I think it should be consistent,